docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+or can be piped into the lowest-priority `cmd_vel` topic.
+
+#### Odometry
+
+Husky publishes odometry information on the `odometry/filtered` topic, as
+[nav_msgs/Odometry messages](http://docs.ros.org/api/nav_msgs/html/msg/Odometry.html).
+These are generated by [ekf_localization_node](http://wiki.ros.org/robot_localization),
+which processes data from several sensor sources using an Extended Kalman filter (EKF).
+This includes data from the wheel encoders and IMU (if available).
+
+| Topic                            | Sources                 | Description                                                        |
+| :------------------------------- | :---------------------- | :----------------------------------------------------------------- |
+| `husky_velocity_controller/odom` | `husky_node`            | Receives motion commands from `twist_mux` based off their priority |
+| `imu/data`                       | `imu_filter_madgwick`   | Joystick teleop input                                              |
+| `odometry/filtered`              | `ekf_localization_node` | Interactive marker teleop input                                    |
+
+Additional odometry information sources can be added to the EKF in
+[localization.yaml](https://github.com/husky/husky/blob/noetic-devel/husky_control/config/twist_mux.yaml).
+
+#### Diagnostics
+
+:::note
+Diagnostics are only applicable to real Husky robots, not simulation.
+:::
+
+Husky provides hardware and software system [diagnostics](http://wiki.ros.org/diagnostics) on the ROS standard
+`/diagnostics` topic. The best way to view these messages is using the
+[rqt_runtime_monitor](http://wiki.ros.org/rqt_runtime_monitor) plugin. Run the `rqt` command, and select
+**Plugins->Robot Tools->Runtime Monitor** from the top menu.
+
+The same information is also published as a
+[husky_msgs\HuskyStatus](http://docs.ros.org/api/husky_msgs/html/msg/HuskyStatus.html) message on the status topic.
+
+### Driving Husky     {#driving-husky}
+
+There are four ways to drive Husky, and each way will work on a physical Husky robot as well as on a simulated Husky.
+
+1. Using the interactive remote controller in RViz. See [Simulation](#simulating-husky).
+2. Using autonomous navigation. See [Navigation](#navigating-husky).
+3. Using the controller for teleoperation. See below.
+4. Publishing ROS messages. See below.
+
+:::warning
+Husky is a heavy, robot capable of reaching high speeds.  Careless driving can cause harm to the operator,
+bystanders, the robot, or other property.  Always remain vigilant, ensure you have a clear line of sight to the
+robot, and operate the robot at safe speeds.
+:::
+
+#### Driving with Remote Controller
+
+:::note
+For instructions on controller pairing, [Pairing the Controller](#pairing-the-controller).
+:::
+
+:::warning
+When familiarizing yourself with Husky's operation, always hold the left button (L1/LB).  Once you are comfortable
```

Comment:
Add a picture of the controller?

https://github.com/jackal/jackal-user-manual/blob/master/Sony_PS4_Controller_Labelled.pdf

Reply:
There is already an image in the 'pairing' section and in the user manual, so I think that should be enough. That particular UM6 topic is also very old and should possibly get removed.